.github/workflows/release-chocolatey.yml

```diff
@@ -0,0 +1,42 @@
+name: Release Chocolatey Package
+on:
+  release:
+    types: [published]
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'Version to release'
+        required: true
+  
+jobs:
+  release:
+    name: Publish to Chocolatey Community
+    runs-on: windows-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Set Version
+        id: release_version
+        run: |
+          if ( "${{ github.event_name }}" -eq "workflow_dispatch" ) {
+            $version = "${{ github.event.inputs.version }}"
+          }
+          else {
+            $version = "${{ github.event.release.tag_name }}" -replace "^v", ""
+          }
+          echo "Setting version to $version"
+          echo "::set-output name=version::$version"
+
+      - name: Make nuspec from the template
+        run: |
+          cd ./deploy/chocolatey
```

Comment:
so these files that we have there, in this directory they stay as they are? and we commit them to repo?

if it is only deployment related, why not having it in `.github` folder? in the end, nobody should ever manually run deployment from local

Reply:
Sure.